.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v10
+      - uses: olafurpg/setup-gpg@v3
```

Comment:
`- uses: olafurpg/setup-gpg@v3` is no longer needed.

Reply:
done